app/models/requests/requestable/item.rb

```diff
@@ -161,7 +161,7 @@ def copy_value
       end
 
       def temp_loc?
-        false
+        ""
```

Comment:
Is changing `false` to `""` necessary?

Reply:
Previously temp_loc? was checking if [self[:temp_loc].present?](https://github.com/pulibrary/orangelight/pull/2945/files#diff-fc52fa5fdd9a5b0b2265bfd72297d0e560cb1099fbe904a25c5e4adcac488655R48). Based on the api response when there is an item it will always return an 'in_temp_library' either true or false. For a null item we won't get anything. This is why I set it to `""`.